build/buildall

```diff
@@ -91,6 +91,17 @@ function versionsFromDistProfile() {
   echo -n $versionStr
 }
 
+function versionsFromReleaseProfiles() {
+  versionStr=$(mvn -B -q -pl . $2 -Dexpression=$1 -DforceStdout org.apache.maven.plugins:maven-antrun-plugin:run@get-buildvers)
+  echo -n $versionStr
+}
+
+function createReleaseProperties() {
+  # Create Scala release properties file
+  echo "Collecting Spark versions involved..."
+  mvn -B -q -pl . $1 -DforceStdout antrun:run@create-release-properties
+}
```

Comment:
Please look for a way to make this maven executions inside pom.

Reply:
The execution is inside the POM as part of the `initialize` phase. This is why we don't need to call this explicitly when doing a `mvn install`. 
The reason why we have to explicitly call this here is because we need to query the pom for setting the `included_buildvers`. Actually since `buildall` only calls the `get-buildvers` once we don't need to create the properties file and we could do away with this in this case. I will have to modify the `get-buildvers` to create the properties file if it doesn't exist.